ReactAndroid/src/main/java/com/facebook/react/views/text/ReactTextShadowNode.java

```diff
@@ -433,6 +444,17 @@ public void setFontStyle(@Nullable String fontStyleString) {
     }
   }
 
+  @ReactProp(name = ViewProps.TEXT_DECORATION_LINE)
+  public void setTextDecorationLine(@Nullable String textDecorationLineString) {
+    mIsUnderlineTextDecorationSet = (
+      "underline".equals(textDecorationLineString) ||
+      "underline line-through".equals(textDecorationLineString));
+    mIsLineThroughTextDecorationSet = (
```

Comment:
Is this a conscious decision or some sort of standard to only support "underline line-though" but not "line-through underline"?